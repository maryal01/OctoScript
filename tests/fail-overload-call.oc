fn overload printout(int i) -> none {
    print("%d", i);
}

fn overload printout(string s) -> none {
    print("%s", s);
}

printout(false);