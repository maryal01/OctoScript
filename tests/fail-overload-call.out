Fatal error: exception Failure("Calling of overloaded function printout doesn't match existing signatures.")